frontend/src/App.tsx

```diff
@@ -45,6 +60,20 @@ function Mayournaise() {
         onSubmit={handleSubmit(onSubmit)}
         className="space-y-4 sm:space-y-6"
       >
+        <div className="flex justify-end mb-2">
+          <button
+            type="button"
+            onClick={randomizeOptions}
+            className="text-indigo-600 hover:text-indigo-800 font-medium text-sm sm:text-base flex items-center bg-indigo-50 py-1 px-3 rounded-md hover:bg-indigo-100 transition-colors"
+          >
+            <svg xmlns="http://www.w3.org/2000/svg" className="h-5 w-5 mr-1" viewBox="0 0 20 20" fill="currentColor">
```

Comment:
this icon looks a bit weird - could you use the magic wand emoji instead, or maybe no image or icon at all